### DISCUSSION TITLE
Code language not showing properly

### DISCUSSION THREAD

**Discussion by ahmedmahmoud321** [community/community#21054] [Projects and Issues]
- I have noteapp repository and it written in java but recently i started to convert it into kotlin put new code not showing in language detector

<img width="296" alt="Screenshot 2022-07-22 071109" src="https://user-images.githubusercontent.com/73083104/180367346-a55df651-7a7b-4d85-9c09-18555cb8aed6.png">

- when i click on kotlin it only show there is 2 classes/files written in kotlin but the Repo containers 22 kotlin file 

<img width="519" alt="Screenshot 2022-07-22 071414" src="https://user-images.githubusercontent.com/73083104/180367642-c29dc986-8add-4d86-ae4e-2a07c7991c94.png">

**Comment by jfullstackdev** [comment]
maybe this one can help 

https://softwareengineering.stackexchange.com/questions/156029/how-does-github-calculate-language-percentage-in-a-repo
